test/features/can_refresh_a_closed_record_test.rb

```diff
@@ -0,0 +1,233 @@
+            {"ContactPersons":[{"ContactInformation":{"ContactMechanisms":[{"Type":"Telephone","Value":"+1 (910) 962-2383"},
+            {"Type":"Email","Value":"binghamf@uncw.edu"}]},"FirstName":"Frederick","LastName":"Bingham","MiddleName":"M","Roles":["Investigator",
+            "Technical Contact"]}],"LongName":"SPURS Data manager, Fred Bingham","Roles":["ARCHIVER"],"ShortName":"NASA/JPL/PODAAC"},
+            {"ContactPersons":[{"ContactInformation":{"ContactMechanisms":[{"Type":"Telephone","Value":"(818) 393-7165"},
+            {"Type":"Email","Value":"podaac@podaac.jpl.nasa.gov"}]},"FirstName":"User","LastName":"Services",
+            "Roles":["Investigator","Technical Contact"]}],"LongName":"Physical Oceanography Distributed Active Archive Center, 
+            Jet Propulsion Laboratory, NASA","Roles":["ARCHIVER"],"ShortName":"NASA/JPL/PODAAC"}],"TemporalKeywords":["1 Day"],
+            "Platforms":[{"Characteristics":[{"DataType":"FLOAT","Description":"Nominal period of a spacecraft`s single revolution on an orbital plane.",
+            "Name":"OrbitPeriod","Unit":"Minutes","Value":"0.0"},{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.",
+            "Name":"InclinationAngle","Unit":"Degrees","Value":"0.0"}],
+            "Instruments":[{"Characteristics":[{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.",
+            "Name":"SwathWidth","Unit":"Kilometers","Value":"0.001"}],"LongName":"Conductivity, Temperature, Depth",
+            "ShortName":"CTD"},{"Characteristics":[{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.",
+            "Name":"SwathWidth","Unit":"Kilometers","Value":"0.001"}],"LongName":"Meteorological Measurement System",
+            "ShortName":"MMS"},{"Characteristics":[{"DataType":"FLOAT",
+            "Description":"Spacecraft angular distance from orbital plane relative to the Equator.","Name":"SwathWidth","Unit":"Kilometers",
+            "Value":"0.001"}],"LongName":"Current Meter","ShortName":"CURRENT METERS"}],"LongName":"SPURS-I WHOI Waveglider",
+            "ShortName":"Waveglider","Type":"In Situ Ocean-based Platforms"}],
+            "ArchiveAndDistributionInformation":{"FileDistributionInformation":[{"Format":"NETCDF","FormatType":"Native"}]}}}]}',
+            headers: {}
+        )
+
+    # stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?revision_id=4")
+    #     .with(
+    #         headers: {'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Accept' => '*/*', 'User-Agent' => 'Ruby'}
+    #     )
+    #     .to_return(status: 200, body: '<?xml version="1.0" encoding="UTF-8"?><results><hits>0</hits><took>32</took></results>', headers: {})
+  end
+
+  describe "GET #refresh" do
+    it "refreshes a closed record and notifies of no new revision" do
```

Comment:
This test should probably be renamed, "refreshes a record and informs user collection has already been ingested.'